src/frequently-asked-questions-about-fsrs.md

```diff
@@ -0,0 +1,39 @@
+# Frequently asked questions about FSRS
+
+***
+
+Q1: Which platforms support FSRS?
+
+A1: FSRS is supported on all platforms: desktop Anki (Windows, Mac, and Linux), AnkiWeb (browser), AnkiMobile (iOS), and AnkiDroid (Android). Make sure that your Anki(Mobile) version is 23.10 or newer, and your AnkiDroid version is 2.17 or newer.
+
+***
+
+Q2: My first interval for 'Good' is 3-5 days, and my first interval for 'Easy' is over a week long! Is this normal?
```

Comment:
either bold (when thought of as UI elements) or double quotes. single quotes aren't used that much.

```suggestion
Q2: My first interval for **Good** is 3-5 days, and my first interval for **Easy** is over a week long! Is this normal?
```